example/pubspec.yaml

```
@@ -1,50 +1,48 @@
 name: example
 description: A new Flutter project.
 
-publish_to: 'none' #Remove this line if you wish to publish in pub.dev
+publish_to: "none" #Remove this line if you wish to publish in pub.dev
 
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.13.0 <3.0.0"
+  sdk: ">=3.3.0 <4.0.0"
```

Seria interessante colocar também a versão do Flutter aqui:
```suggestion
  sdk: ">=3.3.0 <4.0.0"
  flutter: ">=3.19.0"
```